app_dart/lib/src/model/ci_yaml/ci_yaml.dart

```diff
@@ -18,14 +19,34 @@ class CiYaml {
   });
 
   /// The underlying protobuf that contains the raw data from .ci.yaml.
-  final pb.SchedulerConfig config;
+  pb.SchedulerConfig config;
 
   /// The [RepositorySlug] that [config] is from.
   final RepositorySlug slug;
 
   /// The git branch currently being scheduled against.
   final String branch;
 
+  /// Load [currentConfigYaml] (and optionally [totConfigYaml] ) to [pb.SchedulerConfig] and validate the dependency graph.
```

Comment:
The format for dart docs is a single, clear sentence. Then follow up with specific details

```dart
/// Creates [CiYaml] from a [YamlMap].
///
/// If [totConfigYaml] is passed, the validation will verify no new targets have been added that may temporarily break the LUCI infrastructure (such as new prod or presubmit targets).
```